pkg/util/node.go

```diff
@@ -28,7 +28,7 @@ func GetHostname(hostnameOverride string) string {
 	if string(hostname) == "" {
 		// Note: We use exec here instead of os.Hostname() because we
 		// want the FQDN, and this is the easiest way to get it.
```

Comment:
os.Hostname() retrieves the information from /proc/sys/kernel/hostname directly, but on some os dist, uname -n invokes uname syscall, not from that file.